posix.mak

```diff
@@ -31,8 +31,11 @@
 QUIET:=
 
 DEBUGGER=gdb
+GIT_HOME=https://github.com/dlang
+DMD_DIR=../dmd
 
-include osmodel.mak
+$(shell [ ! -d $(DMD_DIR) ] && git clone --depth=1 $(GIT_HOME)/dmd $(DMD_DIR))
```

Comment:
> I assumed that you clone all repos on start?

They're always there, just cleaned/reset between runs. Not sure why you'd get an error there.